rich/_console.py

```diff
@@ -0,0 +1,151 @@
+            align-items: center;
+            background-color: {theme_background_color};
+            border-radius: 14px;
+            outline: 1px solid #484848;
+        }}
+        .{classes_prefix}-terminal:after {{
+            position: absolute;
+            width: 100%;
+            height: 100%;
+            content: '';
+            border-radius: 14px;
+            background: rgb(71,77,102);
+            background: linear-gradient(90deg, #804D69 0%, #4E4B89 100%);
+            transform: rotate(-4.5deg);
+            z-index: -1;
+        }}
+        .{classes_prefix}-terminal-header {{
+            position: relative;
+            width: 100%;
+            background-color: #2e2e2e;
+            margin-bottom: 12px;
+            font-weight: bold;
+            border-radius: 14px 14px 0 0;
+            color: {theme_foreground_color};
+            font-size: 18px;
+            box-shadow: inset 0px -1px 0px 0px #4e4e4e,
+                        inset 0px -4px 8px 0px #1a1a1a;
+        }}
+        .{classes_prefix}-terminal-title-tab {{
+            display: inline-block;
+            margin-top: 14px;
+            margin-left: 124px;
+            font-family: sans-serif;
+            padding: 14px 28px;
+            border-radius: 6px 6px 0 0;
+            background-color: {theme_background_color};
+            box-shadow: inset 0px 1px 0px 0px #4e4e4e,
+                        0px -4px 4px 0px #1e1e1e,
+                        inset 1px 0px 0px 0px #4e4e4e,
+                        inset -1px 0px 0px 0px #4e4e4e;
+        }}
+        .{classes_prefix}-terminal-traffic-lights {{
+            position: absolute;
+            top: 24px;
+            left: 20px;
+        }}
+        .{classes_prefix}-terminal-body {{
+            line-height: {line_height}px;
+            padding: 14px;
+        }}
+        {stylesheet}
+    </style>
+    <foreignObject x="0" y="0" width="100%" height="100%">
+        <body xmlns="http://www.w3.org/1999/xhtml">
+            <div class="{classes_prefix}-terminal-wrapper">
+                <div class="{classes_prefix}-terminal">
+                    <div class="{classes_prefix}-terminal-header">
+                        <svg class="{classes_prefix}-terminal-traffic-lights" width="90" height="21" viewBox="0 0 90 21" xmlns="http://www.w3.org/2000/svg">
+                            <circle cx="14" cy="8" r="8" fill="#ff6159"/>
+                            <circle cx="38" cy="8" r="8" fill="#ffbd2e"/>
+                            <circle cx="62" cy="8" r="8" fill="#28c941"/>
+                        </svg>
+                        <div class="{classes_prefix}-terminal-title-tab">{title}</div>
+                    </div>
+                    <div class="{classes_prefix}-terminal-body">
+                        {code}
+                    </div>
+                </div>
+            </div>
+        </body>
+    </foreignObject>
+</svg>
+"""
+
+
+def _svg_hash(svg_main_code: str) -> str:
```

Comment:
This probably doesn't belong in a file full of constants.

Perhaps it should be a closure? If we don't need to access it outside of the export svg function.

Can you please add a docstring for this. We use the google format for docstrings. I have a vscode plugin which can partially generate them.